content/c-sharp/concepts/math-functions/terms/acos/acos.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: '.Acos()'
+Description: 'Returns the inverse cosine of the argument.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arithmetic'
+  - 'Numbers'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`Math.Acos()`** method returns the absolute value of the inverse cosine of the argument, measured in radians. In other words: It returns the positive angle whose cosine is the specified number.
+
+## Syntax
+
+```pseudo
+Math.Acos(x);
+```
+
+The method takes one parameter `x` of the type `double`. It will return a positive value of type `double` except in the following cases:
+-If `x` is less than -1, it will return `NaN` (not a number).
+-If `x` is greater than 1, it will return `NaN`.
+-If `x` is `NaN`, it will return `NaN`.
```

Comment:
```suggestion
- If `x` is `NaN`, it will return `NaN`.
```

Reply:
Hi @caupolicandiaz , I made the changes :)